docs/sources/dashboards/dashboard-public/index.md

```diff
@@ -8,6 +8,7 @@ labels:
     - enterprise
     - oss
 title: Public dashboards
+description: Share your Grafana dashboards with anyone using public dashboards
```

Comment:
"using public dashboards" might be mistakenly interpreted as describing the "anyone". Maybe something like this instead:
"Make your Grafana dashboards public and share it with anyone"

Reply:
Ah makes sense!